osidb/serializer.py

```diff
@@ -604,7 +604,10 @@ class Meta:
 
     def to_representation(self, instance):
         representation = super().to_representation(instance)
-        if representation["pgh_diff"]:
+        if (
+            isinstance(representation["pgh_diff"], dict)
+            and "last_validated_dt" in representation["pgh_diff"]
+        ):
             representation["pgh_diff"].pop("last_validated_dt")
```

Comment:
Is it guaranteed that `last_validated_dt` is there? I am not too familiar with the history and whether the record is always a full model or just a subset but I mean that it is a pretty new attribute. Evetually `.pop("last_validated_dt", None)` should make sure it does not result in a `KeyError`.

Reply:
It is covered above in the condition:
```python
and "last_validated_dt" in representation["pgh_diff"]
```

Reply:
No idea why I did not see it :smile: never mind then...